serve-web/src/Service/ReportService.php

```diff
@@ -59,18 +66,23 @@ public function generateCsv()
 
         fclose($file);
 
-        return new File('/tmp/orders.csv');
+        return new File("/tmp/orders-served-$today.csv");
     }
 
     /**
-     *  Get orders that have been served into Sirius
+     *  Get orders that have been served into Sirius for the last 4 weeks
      *
-     * @return \App\Entity\Order[]
+     * @return Order[]
      */
     public function getOrders()
     {
+        $today = (new DateTime('now'))->format('Y-m-d');
+        $minus4Weeks = (new DateTime('now'))->modify('-4 weeks')->format('Y-m-d');
```

Comment:
Hardcoded for now, but for future work we should we should include a filter on the UI where the user can specify the date range, and if no date range given, then default to last 4 weeks